crunchy-cli-core/src/archive/command.rs

```diff
@@ -86,6 +86,12 @@ pub struct Archive {
     #[arg(value_parser = MergeBehavior::parse)]
     pub(crate) merge: MergeBehavior,
 
+    #[arg(
+        help = "If in auto merge mode. Milliseconds if which audio track lengths are within each other, the audio merge method is used"
+    )]
```

Comment:
The help message is a bit confusing. Something like `If the merge behavior is 'auto', only download multiple video tracks if their length difference is higher than the given milliseconds` is more clear